components/ExercisePreview/index.js

```diff
@@ -0,0 +1,47 @@
+import styled from "styled-components";
+export default function ExercisePreview({ exercise }) {
+  return (
+    <ExerciseCard image={exercise.imageUrl}>
```

Comment:
```suggestion
    <ExerciseCard $image={exercise.imageUrl}>
```
!!! :D Fast übersehen, gut dass es Fehlermeldungen gibt!